### PR TITLE
fix(workflow): Change `<ButtonBar>` children to be "primary" when active

### PR DESCRIPTION
**Pull request by billyvg**
This is the proper way to show "active" state. This logic should probably be moved into `<ButtonBar>`

### CODE REVIEW

**Comment by billyvg** [comment]
@priscilawebdev I was wrong about the styling 🙊

File: src/sentry/static/sentry/app/views/alerts/list/index.tsx [hunk 1]
```diff
@@ -237,21 +243,21 @@ class IncidentsListContainer extends React.Component<Props> {
                 <Button
                   to={{pathname, query: openIncidentsQuery}}
                   size="small"
-                  className={'btn' + (status === 'open' ? ' active' : '')}
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/alerts/list/index.tsx]: You still need the className of `active` to get correct z-index stacking.

**Review by markstory — APPROVED** [review]
Looks good. Having `ButtonBar` change child Button colors based on activeness seems like it could backfire as not all button bar will want the active button to be purple. I'm not sure what a better API would be at this point.

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Pull request merged**